lib/dry/types/coercions/form.rb

```diff
@@ -10,13 +10,10 @@ module Form
         TRUE_VALUES = %w[1 on  t true  y yes].freeze
         FALSE_VALUES = %w[0 off f false n no].freeze
         BOOLEAN_MAP = ::Hash[TRUE_VALUES.product([true]) + FALSE_VALUES.product([false])].freeze
+        EMPTY_STRING = ''.freeze
```

Comment:
Hah literally yesterday I looked at these coercions and noticed this duplicated empty string allocations and thought "ugh, this should've been a constant" :)